src/com/ensa/gi4/controller/AuthenticationController.java

```diff
@@ -0,0 +1,82 @@
+package com.ensa.gi4.controller;
+
+import com.ensa.gi4.modele.User;
+import com.ensa.gi4.service.api.AuthenticationService;
+import com.ensa.gi4.utils.PasswordEncryptingUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Controller;
+import java.util.Scanner;
+
+@Controller("authenticationController")
+public class AuthenticationController {
+
+    @Value("${menu.incorrect.credentials}")
+    String incorrectCredentialsMenu;
+
+    @Autowired
+    User authenticatedUser;
+    @Autowired
+    AuthenticationService authenticationService;
+    @Autowired
+    @Qualifier("adminController")
```

Comment:
Qualifer is not mandatory, the name of the service is understood from the variable name